src/State/AuthFlow/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+// TODO: load from the env config once implemented
+const COGNITO_URL =
```

Comment:
Yeah we should implement env variable file. I will open a task for this

Reply:
the story is already in the sprint! https://edma.atlassian.net/browse/KIR-330